index.html

```diff
@@ -0,0 +1,328 @@
+                            website Look through some of my work 
+                            and experience! If you like what you 
+                            see and have a project you need coded, 
+                            don’t hesitate to contact me.
+                        </p>
+                        <div class="btn-holder">
+                            <button class="btn">Get My Resume</button>
+                        </div>
+                        
+                        <img class="about-me-banner" src="/assets/images/About Me Background.svg" alt="Aboutmebanner">
+                        
+                </article>
+                <div class="divider"></div>
+            
+                <article class="skills-container">
+                    <article class="skills1">
+                        <article class="skills1-top">
+                            <img src="/assets/images/Hexagon.svg" alt="hexagon">
+                            <h1>Languages</h1>
+                        </article>
+                        <article class="list-items">
+                            <ul class="list-item">
+                                <li><a href="#">Javascript</a></li>
+                                <li><a href="#">Ruby</a></li>
+                                <li><a href="#">HTML</a></li>
+                                <li><a href="#">css</a></li>
+                            </ul>
+                        </article>
+                    </article>
+                    <article class="skills2">
+                        <article class="skills2-top">
+                            <img src="/assets/images/Square.svg" alt="Square">
+                            <h1>Frameworks</h1>
+                        </article>
+                        <article class="list-items">
+                            <ul class="list-item">
+                                <li><a href="#">React.js</a></li>
+                                <li><a href="#">Ruby on Rails</a></li>
+                                <li><a href="#">RSpec</a></li>
+                                <li><a href="#">Capybara</a></li>
+                            </ul>
+                        </article>
+                        
+                    </article>
+
+                    <article class="skills2">
+                        <article class="skills2-top">
+                            <img src="/assets/images/Circle.svg" alt="Circle">
+                            <h1>Skills</h1>
+                        </article>
+                        <article class="list-items">
+                            <ul class="list-item">
+                                <li><a href="#">Database Management</a></li>
+                                <li><a href="#">Version Control</a></li>
+                                <li><a href="#">CLI</a></li>
+                                <li><a href="#">API Design</a></li>
+                            </ul>
+                        </article>
```

Comment:
> - [ ] Great job so far👍. I noticed that skills category alignment does not stick with the Figma design as much as possible. Please could you kindly fix that? This will create a better user experience and feel for your website.😊 Also, consider the heading font weight to match the design.👍
> 
> Figma design:
> 
> ![language1](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/40009354/f5109040-b17e-4362-9a6b-b72932b1fa27)
> 
> Your design:
> 
> ![language](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/40009354/09c48359-bd5a-4d71-b2cf-fc72c7e63214)
>